docs/release-notes/8.14.asciidoc

```diff
@@ -5,6 +5,27 @@
 [[release-notes-8.14.3]]
 === 8.14.3
 
+[discrete]
+[[known-issue-8.14.3]]
+==== Known issues
+
+// tag::known-issue-14686[]
+[discrete]
+.{elastic-endpoint} does not properly populate the `user.name` field 
```

Comment:
```suggestion
.{elastic-endpoint} does not properly populate the `user.name` field in security events.
```

`user.name` exists in a lot of other events, so omitting that final prepositional phrase is ambiguous.

Reply:
Making a minor adjustment to the suggestion above (and everything following) to remove the period at the end of the title.